gix-diff/src/rewrites/tracker.rs

```diff
@@ -561,10 +561,10 @@ mod diff {
         type Out = usize;
 
         fn process_change(&mut self, before: Range<u32>, _after: Range<u32>) {
-            self.removed_bytes = self.input.before[before.start as usize..before.end as usize]
+            self.removed_bytes += self.input.before[before.start as usize..before.end as usize]
```

Comment:
It's amazing that it's just this!

Reply:
I was genuinely surprised myself, but it makes all the tests pass and the old tests continue passing.  We will likely find more over time, but this will definitely improve things in the short term!